ospfd/ospf_lsa.h

```diff
@@ -230,7 +230,7 @@ extern struct timeval msec2tv(int);
 extern int get_age(struct ospf_lsa *);
 extern uint16_t ospf_lsa_checksum(struct lsa_header *);
 extern int ospf_lsa_checksum_valid(struct lsa_header *);
-extern int ospf_lsa_refresh_delay(struct ospf_lsa *);
+extern int ospf_lsa_refresh_delay(struct ospf *, struct ospf_lsa *);
```

Comment:
please put variable names in.  While the surrounding code doesn't, we made a conscious choice to require all new code to have variable names in function declarations.  I've been going through and fixing up missing header data as I run across it.  I'll get ospf_lsa.h fixed up here shortly

Reply:
Sounds good. This must a relatively new convention or at least a not well-known one.